services/brig/test/integration/API/User/Auth.hs

```diff
@@ -279,6 +345,92 @@ testLimitRetries (Just conf) brig = do
     login brig (defEmailLogin email) SessionCookie !!! const 200 === statusCode
 
 
+-------------------------------------------------------------------------------
+-- LegalHold Login
+
+testRegularUserLegalHoldLogin :: Brig -> Http ()
+testRegularUserLegalHoldLogin brig = do
+    -- Create a regular user
+    uid <- userId <$> randomUser brig
+    -- fail if user is not a team user
+    legalHoldLogin brig (LegalHoldLogin uid (Just defPassword) Nothing) PersistentCookie !!! do
+        const 403 === statusCode
+
+testTeamUserLegalHoldLogin :: Brig -> Galley -> Http ()
+testTeamUserLegalHoldLogin brig galley = do
+    -- create team user Alice
+    (alice, tid) <- createUserWithTeam brig galley
+    now <- liftIO getCurrentTime
+    -- fail if legalhold isn't activated yet for this user
+    legalHoldLogin brig (LegalHoldLogin alice (Just defPassword) Nothing) PersistentCookie !!! do
+        const 403 === statusCode
+
+    putLegalHoldEnabled tid LegalHoldEnabled galley -- enable it for this team
+    _rs <- legalHoldLogin brig (LegalHoldLogin alice (Just defPassword) Nothing) PersistentCookie
```

Comment:
Fail again here at the moment LH is enabled for team, but not for member?

Reply:
That should be tested as part of galley's legal hold tests (if it's not, it should get added there, but it's not related to this PR)